index.js

```diff
@@ -83,20 +85,44 @@ export default class SlimSlider{
     this.slideCount = Math.ceil(this.slides.length / this.options.itemsPerSlide);
     this.slideWidth = this.parent.offsetWidth;
     this.itemWidth = this.parent.offsetWidth / this.options.itemsPerSlide;
+    this.autoPlay = this.options.autoPlay;
+    this.autoPlayTimer = this.options.autoPlayTimer;
     this.initDom();
     this.options.showPointers && this.createPagination();
     this.options.showThumbnails && this.createThumbs();
     this.options.showButtons && this.createButtons();
     this.initGesture();
     this.registerListeners();
-    dispatchEvent(this.parent, 'after.slim.init', { current:this.current })
+    dispatchEvent(this.parent, 'after.slim.init', { current:this.current });
+
+    if (this.autoPlay) {
+      this.initAutoPlay();
+    }
+  }
+
+  initAutoPlay() {
+    this.autoplayerInterval = setInterval(_=> {
+      this.goToNext();
+    }, this.autoPlayTimer);
+  }
+
+  removeAutoPlay() {
+    if (this.autoPlay) {
+      this.slider.classList.remove('autoplay');
+      this.autoPlay = false;
+      clearInterval(this.autoplayerInterval);
+    }
   }
+
   /**
    * Prepares the current slider dom with neccessary data.
    */
   initDom(){
     if(!this.slider || !this.sliderWrapper){
       this.slider = create('div', {class:'slim-slides'})
+      if (this.autoPlay) {
+        this.slider.classList.add('autoplay');
```

Comment:
suggestion: this.autoPlay && this.slider.classList.add('autoplay');

Reply:
👍